PWGHF/TableProducer/HFCandidateCreatorBPlus.cxx

```diff
@@ -114,8 +114,8 @@ struct HfCandidateCreatorBplus {
       auto prong1TrackParCov = getTrackParCov(prong1);
       auto collision = prong0.collision();
 
-      // LOGF(info, "All track: %d (prong0); %d (prong1)", candidate.index0().globalIndex(), candidate.index1().globalIndex());
-      // LOGF(info, "All track pT: %f (prong0); %f (prong1)", prong0.pt(), prong1.pt());
+      // LOGF(INFO, "All track: %d (prong0); %d (prong1)", candidate.index0().globalIndex(), candidate.index1().globalIndex());
+      // LOGF(INFO, "All track pT: %f (prong0); %f (prong1)", prong0.pt(), prong1.pt());
```

Comment:
Why did you change these? See https://github.com/AliceO2Group/O2Physics/pull/102/files